Makefile

```diff
@@ -1,5 +1,18 @@
-# Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+# The semver version number which will be used as the Docker image tag
+# Defaults to the output of git describe.
+VERSION ?= $(shell git describe --tags --dirty --always)
+
+# Docker image name parameters
+DOCKER_PREFIX ?= quay.io/cert-manager/signer-ca/
```

Comment:
I'm not sure if nesting like this is valid on quay.io.. this would result in images like `quay.io/cert-manager/signer-ca/controller:abcdefg`.

Maybe not an issue for now given we aren't pushing these images at the moment 😄

Reply:
Good point. I made it `quay.io/cert-manager/signer-ca-controller:abcdefg` instead.